man/rgbasm.5

```diff
@@ -328,76 +328,100 @@ with a non-zero constant as either operand will be constant 1, even if the other
 returns 1 if the operand was 0, and 0 otherwise.
 Even a non-constant operand with any non-zero bits will return 0.
 .Ss Fixed-point expressions
-Fixed-point numbers are basically normal (32-bit) integers, which count fractions instead of whole numbers.
-They offer better precision than integers but limit the range of values.
-By default, the upper 16 bits are used for the integer part and the lower 16 bits are used for the fraction (65536ths).
-The default number of fractional bits can be changed with the
+Fixed-point numbers are basically normal integers, but conceptually they contain a dot at a fixed location\(emhence the name.
+This gives them an increased precision, at the cost of a smaller range, while remaining far cheaper to manipulate than floating-point numbers.
+.Pp
+By default,
+.Nm
+uses the precision specified by the
 .Fl Q
-command-line option.
-You can also specify a precise fixed-point value by appending a
+command-line option and/or by
+.Ic OPT Q
+.Pq see Sx Changing options while assembling ;
+the default precision is 16 bits (meaning the lower 16 bits are used for the fractional part, so the numbers count in 65536ths).
+The precision of a literal can be overridden by appending a
 .Dq q
-to it followed by the number of fractional bits, such as
-.Ql 12.34q8 .
-.Pp
-Since fixed-point values are still just integers, you can use them in normal integer expressions.
-Some integer operators like
-.Sq +
-and
-.Sq -
-don't care whether the operands are integers or fixed-point.
-You can easily truncate a fixed-point number into an integer by shifting it right by 16 bits.
-It follows that you can convert an integer to a fixed-point number by shifting it left.
-.Pp
-The following functions are designed to operate with fixed-point numbers:
+followed by the number of fractional bits:
```

Comment:
I think "By default" is very misleading here, when the actual *default* is further below.

Suggestion:
> The default precision of all fixed-point numbers is 16 bits, meaning the lower 16 bits are used for the fractional part; so they count in 65536ths of 1.0.
> This precision can be changed with the Q command-line option, or by OPT Q (see "Changing options while assembling").
> An individual fixed-point literal can have its own precision by appending a `q` followed by the number of fractional bits: for example,

Reply:
(To be fair, this was also a problem with the original text :P)